docs/TypeConverters/index.md

```diff
@@ -0,0 +1,148 @@
+# Type Converters
+
+Provides a unified way of converting types of values to other types, as well as for accessing values and supporting custom serialzation.
```

Comment:
The original sentence felt like a little stroke. Not sure what you wanted to say. I propose maybe this (?):
> Provides a unified way of converting values between different variable types and supports custom serialization using CF_IO.

Reply:
That's what I get from basically copying the C# docs